gitlab/resource_gitlab_pipeline_schedule_variable.go

```diff
@@ -88,7 +88,8 @@ func resourceGitlabPipelineScheduleVariableRead(d *schema.ResourceData, meta int
 		}
 	}
 	if !found {
-		return fmt.Errorf("PipelineScheduleVariable %s can not be found", pipelineVariableKey)
+		log.Printf("[DEBUG] pipeline schedule variable not found %s/%d/%s", project, scheduleID, pipelineVariableKey)
```

Comment:
Same comment as above. This `gitlab_pipeline_schedule_variable` resource does not have an importer.

Reply:
same comment as above as well, let me know!